docs/contributor/api-server.md

```
@@ -133,7 +133,7 @@ func main() {
 - Server configured with production defaults
 - Graceful shutdown on SIGINT/SIGTERM
 
-**Initialization Flow:**
+#### Initialization Flow
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)